bootstrap.m

```diff
@@ -1,13 +1,13 @@
 function bootstrap
   
   % GENERIC SOLVER INTERFACE
-  addpath(genpath('../lib/yalmip'))
```

Comment:
Let's leave both in meanwhile, if it complains then we can find a way of doing an 'if' statement